doc/security/specs/secure_boot/index.md

```diff
@@ -176,8 +190,10 @@ manifest format is required to support:
 [ibex-epmp]: https://ibex-core.readthedocs.io/en/latest/03_reference/pmp.html
 [identities-keys]: {{< relref "/doc/security/specs/identities_and_root_keys" >}}
 [key-manager]: {{< relref "/hw/ip/keymgr/doc" >}}
+[manifest-format]: {{< relref "/sw/device/silicon_creator/rom_ext/docs/manifest" >}}
 [mask-rom-epmp]: {{< relref "/sw/device/silicon_creator/mask_rom/docs/memory_protection" >}}
```

Comment:
This cross-reference is useful along `ibex-epmp`.

Reply:
Thanks for pointing it out, I'll add that link to the text about ePMP :slightly_smiling_face: